src/multi_vector_simulator/E3_indicator_calculation.py

```diff
@@ -876,14 +879,49 @@ def equation_onsite_energy_matching(
     return onsite_energy_matching
 
 
-def equation_co2_emissions(dict_values):
-    co2_emissions = 0
-    for asset in dict_values[ENERGY_PRODUCTION]:
-        co2_emissions += (
-            dict_values[ENERGY_PRODUCTION][asset]["total_aggregated_flow"][VALUE]
-            * dict_values[ENERGY_PRODUCTION][asset]["emissionFactor"][VALUE]
-        )
-    return co2_emissions
+def add_total_emissions(dict_values):
+    r"""
+    Calculates the total emission of the energy system in kg/a and kg/kWheleq and adds KPI to `dict_values`.
+
+    Parameters
+    ----------
+    dict_values: dict
+        All simulation inputs and results
+
+    Returns
+    -------
+    None
+        updated `dict_values` with total emissions KPI, unit: kg/a and kg/kWheleq
```

Comment:
If you mention specificly the constant variables that the function adds to dict_values, it is easier to write the pytest/follow the dict updates